src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -128,6 +128,8 @@ private static bool PublishProjectContext(ProjectContext context, string outputP
                 PublishFiles(export.NativeLibraries, outputPath, nativeSubdirectories);
             }
 
+            CopyContents(context, outputPath);
```

Comment:
Removed *some* of the duplicated code. There's still some duplication because publish doesn't use `LibraryAsset`. I'll put a bug to consolidate everything.